recipe/meta.yaml

```diff
@@ -13,8 +13,9 @@ source:
   sha256: 50b2cca032a4986d8b603a282847c50f7571c76284a9b0a229938c969e25eeda
 
 build:
+  noarch: generic
```

Comment:
This is strange that the bot is adding a second noarch field. I opened Issue https://github.com/regro/cf-scripts/issues/425 to inquire about this